### DISCUSSION TITLE
Compatibility  MIUI 12 para POCO, baseado no Android 10 POCO Launcher 2.0

### DISCUSSION THREAD

**Discussion by thifs** [Navid200/xDrip#109] [Libre]
i wandering to get a  Xiaomi that uses this operacional sistem. Does xdrip and nightscout works good with this? what´s version ? if not, which smartphone works better with it ?
how about android 11 ?

**Comment by Navid200** [comment]
It may help to know what sensor you use.

**Reply by thifs** [reply]
freestyle libre 1 !

**Comment by Navid200** [comment]
We have discussions now opened in the main repository:
https://github.com/NightscoutFoundation/xDrip/discussions

I am looking for a way to transfer the open discussions from here to there. But, I'm not sure I'll succeed. In that case, it may be easier to open new discussions there.
